### PR TITLE
uid: update form placeholders

### PR DESCRIPTION
**Pull request by oliver-sanders**
**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

Response to: https://github.com/cylc/cylc-ui/pull/876#pullrequestreview-850642656

### CODE REVIEW

File: src/components/graphqlFormGenerator/components/vuetify.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default {
     },
     NamespaceIDGlob: {
       is: VTextField,
-      placeholder: 'name[.cycle][:status]',
+      placeholder: 'cycle[/task][:status]',
```
Comment (MetRonnie) [inline src/components/graphqlFormGenerator/components/vuetify.js:157]: Should this be?
```suggestion
      placeholder: 'cycle/task[:status]',
```

Comment (oliver-sanders) [reply src/components/graphqlFormGenerator/components/vuetify.js:157]: The task is now optional e.g. `cylc hold 'workflow//*'`

**Review by MetRonnie — APPROVED** [review]
I noticed if the user accidentally specifies just the name:



Then a somewhat confusing warning appears in the log
```
INFO - Processing 1 queued command(s)
WARNING - No active tasks in the family '*' matching: foo
INFO - Command succeeded with 1 warning(s): hold(['foo'])
```
But otherwise, all good

**Comment by oliver-sanders** [comment]
> I noticed if the user accidentally specifies just the name:

Investigating....

**Comment by oliver-sanders** [comment]
https://github.com/oliver-sanders/cylc-flow/pull/new/universal-id.explicit-warnings

**Pull request merged**